includes/kp-functions.php

```diff
@@ -340,7 +340,7 @@ function kp_is_country_available( $country ) {
 	$country = strtolower( $country );
 	if ( empty( $available_countries ) ) {
 		// See if the country has values saved from the old settings, before the available countries setting was added.
-		$testmode = $settings['testmode'] ?? 'no';
+		$testmode = 'yes' === $settings['testmode'] ?? 'no';
 		$prefix   = $testmode ? 'test_' : '';
```

Comment:
I think `$testmode` will always evaluate to true here since `'no'` is a truthy, thus the `$prefix` will always be `test_`.